variables.tf

```diff
@@ -9,3 +13,257 @@ variable "tags" {
   description = "User-Defined tags"
   type        = map(string)
 }
+
+########################################
+# Bitbucket Vars
+########################################
+
+variable "admin_email" {
+  description = "email address for administrator"
+  type        = string
+}
+
+variable "admin_password" {
+  description = "initial password to use for administrator"
+  type        = string
+}
+
+variable "data_volume_size" {
+  description = "Size in GB of the data volume"
+  type        = number
+}
+
+variable "dns_prefix" {
+  default     = null
+  description = "the hostname that will be used for bitbucket. This will be combined with the domain in `zone_id` or the value of `domain_name` to form the base url."
+  type        = string
+}
+
+variable "domain_name" {
+  default     = null
+  description = "domain name, which is only used if `zone_id` is not specified to compute the base url"
+  type        = string
+}
+
+variable "license_key" {
+  default     = ""
+  description = "Bitbucket license key (optional, must be a single line)"
```

Comment:
Would be great to add a validation block here

Reply:
But not today :(